SDAPackageWithVectorField/setup.py

```diff
@@ -0,0 +1,33 @@
+# This is the setup file for pip
+from setuptools import setup, find_packages
+import os, sys
+from os import path
+
+setup(
+    name = 'SDAWtihVectorField',
+
+    version = '0.0.2',
+
+    description = 'Object detection and Avoidance for SUAS Competition',
+
+    url = 'https://github.com/FlintHill/SUAS-Competition',
+
+    author = 'Vale Tolpegin',
```

Comment:
I am not the author of this package. You should use your name and email address.